### ISSUE TITLE
remove empty intersections 

### ISSUE THREAD

**Issue by mayer-lab** [krassowski/complex-upset#73]
Hi, how can empty intersections be removed? Like the one in your example R-Vignette, chapter 0 with the size 205:
<img width="527" alt="example" src="https://user-images.githubusercontent.com/36899445/101190713-0ba0aa00-3659-11eb-9432-fbb5c0a5715f.png">

**Comment by krassowski** [comment]
Thank you for asking! Yes, you can request that `min_degree=1`, e.g.:

```R
upset(movies, genres, name='genre', width_ratio=0.1, min_degree=1)
```

There was a "bug" in calculation of degree for empty intersection, but I just fixed it and released the fix with version 0.7.1. If you wish to upgrade just install the package again with:

```
if(!require(devtools)) install.packages("devtools")
devtools::install_github("krassowski/complex-upset")
```

![image](https://user-images.githubusercontent.com/5832902/101198139-e44adc80-365a-11eb-83e5-83f8b2372820.png)

**Comment by mayer-lab** [comment]
Thanks a lot !
